resolwe/flow/serializers/entity.py

```diff
@@ -18,6 +18,7 @@ class Meta(CollectionSerializer.Meta):
         fields = CollectionSerializer.Meta.fields + (
             'collections',
             'descriptor_completed',
+            'type',
```

Comment:
Don't forget about the filter. We need to be able to query for entities of given type.